src/imitation/data/rollout.py

```diff
@@ -358,9 +364,11 @@ def generate_trajectories(
     #
     # To start with, all environments are active.
     active = np.ones(venv.num_envs, dtype=bool)
+    assert isinstance(obs, np.ndarray)
```

Comment:
Add comment explaining this is because we don't support dict/tuple observations?